tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -60,26 +61,31 @@ internal class LiveDebugger
             _lineProbeResolver = lineProbeResolver;
             _debuggerSink = debuggerSink;
             _probeStatusPoller = probeStatusPoller;
-            _remoteConfigurationManager = remoteConfigurationManager;
+            _subscriptionManager = remoteConfigurationManager;
             _configurationUpdater = configurationUpdater;
             _dogStats = dogStats;
             _unboundProbes = new List<ProbeDefinition>();
-            Product = new LiveDebuggerProduct();
             ServiceName = serviceName;
+            _subscription = new Subscription(
+                (updates, removals) =>
+                {
+                    AcceptAddedConfiguration(updates.Values.SelectMany(u => u).Select(i => new NamedRawFile(i.Path, i.Contents)));
+                    AcceptRemovedConfiguration(removals.Values.SelectMany(u => u));
+                    return new List<ApplyDetails>(0);
```

Comment:
Not a biggie, but could we use `Array.Empty` or equivalent here to spare the needless allocation?

Reply:
fixed in fc98cce9a7f776057ad7507bbf7e100c5a2f4f23 thanks!